src/bin/ion/commands/beta/mod.rs

```diff
@@ -39,7 +36,7 @@ pub fn run(_command_name: &str, matches: &ArgMatches<'static>) -> Result<()> {
             "The requested beta command ('{}') is not supported and clap did not generate an error message.",
             command_name
         );
-        unreachable!(message);
+        unreachable!("{}", message);
```

Comment:
Noticed you fixed this already in #21. Apologies for including too many things in this CR, I'll avoid this in the future. Would it be better if I send a new CR rebased on the main branch with only the commits to add the driver test and workflow?

Reply:
> Would it be better if I send a new CR rebased on the main branch with only the commits to add the driver test and workflow?

That would be really helpful if you don't mind!